src/Test/L0/TestHostContext.cs

```diff
@@ -388,6 +391,34 @@ public ContainerInfo CreateContainerInfo(Pipelines.ContainerResource container,
             return containerInfo;
         }
 
+        public Mock<IExecutionContext> CreateExecutionContext()
+        {
+            var context = new Mock<IExecutionContext>();
+            context.Setup(x => x.Error(It.IsAny<Exception>())).Callback( (Exception ex) => {
+                context.Object.AddIssue(new Issue() { Type = IssueType.Error, Message = ex.Message });
+                //Console.WriteLine(ex.ToString());
```

Comment:
Can we kill all of the `Console.Writeline` calls here?

Reply:
yup, I meant to do this already.